packages/site/src/pages/Bounties/index.js

```diff
@@ -51,7 +51,7 @@ const DEFAULT_QUERY_PAGE = 1;
 
 const TYPES = {
   bounties: "bounties",
-  childBounties: "child-bounties",
+  childBounties: "childbounties",
```

Comment:
I know this is not pretty, but its a legacy issue

if we want to modify it to camel style, we will have bunch of work todo, too much for the moment

Reply:
https://github.com/opensquare-network/dotreasury/blob/main/packages/site/src/pages/Bounties/BountiesTable.js#L78